eq-author/src/App/QuestionnaireDesignPage/index.test.js

```diff
@@ -106,6 +112,41 @@ describe("QuestionnaireDesignPage", () => {
 
       expect(mockHandlers.onAddPage).toHaveBeenCalledWith(section.id, 1);
     });
+
+    it("should be able to add a page at the start when on a section", () => {
+      wrapper.setProps({
+        match: {
+          params: {
+            questionnaireId: questionnaire.id,
+            entityName: "section",
+            entityId: section.id,
+          },
+        },
+      });
+
+      wrapper.find(`[data-test="side-nav"]`).simulate("addPage");
```

Comment:
```suggestion
      wrapper.find(NavigationSidebar).simulate("addPage");
```

_Remember to import NavigationSidebar.  Also let's fix other occurrences of this throughout this file_